zebra-chain/Cargo.toml

```diff
@@ -39,7 +39,8 @@ proptest-derive = { version = "0.3.0", optional = true }
 
 # ZF deps
 displaydoc = "0.2.0"
-ed25519-zebra = "2"
+# ed25519-zebra = "2"
+ed25519-zebra = {git = "https://github.com/ZcashFoundation/ed25519-zebra", rev = "856c96500125e8dd38a525dad13dc64a0ac672cc"}
```

Comment:
We should maybe follow up this and the `redjubjub` in a new ticket. I will prefer zebra dependencies to be stable.

Reply:
https://github.com/ZcashFoundation/ed25519-zebra/issues/45

Reply:
Also added to https://github.com/ZcashFoundation/zebra/issues/1086